pkgs/by-name/de/deej/package.nix

```
@@ -0,0 +1,48 @@
+{ lib,
+  buildGoModule,
+  fetchFromGitHub,
+  pkg-config,
+  libappindicator-gtk3,
+  webkitgtk
+}:
+
+buildGoModule rec {
+  pname = "deej";
+  version = "0.9.10";
+  rev = "7567f83f2773e40cb950101d0784c96b1e566509";
+
+  src = fetchFromGitHub {
+    owner = "omriharel";
+    repo = "deej";
+    rev = rev;
+    hash = "sha256-T6S3FQ9vxl4R3D+uiJ83z1ueK+3pfASEjpRI+HjIV0M=";
+  };
+
+  vendorHash = "sha256-1gjFPD7YV2MTp+kyC+hsj+NThmYG3hlt6AlOzXmEKyA=";
+
+  ldflags = [
+    "-X main.versionTag=v${version}"
+    "-X main.gitCommit=${rev}"
+    "-X main.buildType=nix-release"
+    "-s"
+    "-w"
+  ];
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libappindicator-gtk3 webkitgtk ];
+
```

```suggestion
  postInstallHook = "mv $out/bin/cmd $out/bin/deej
```

install under a less generic name

---

It looks like this should be a `postInstall` not `postInstallHook`, at least that's whats working locally.

```suggestion
postInstall = "mv $out/bin/cmd $out/bin/deej";
```